python_modules/libraries/dagster-datadog/dagster_datadog/resources.py

```diff
@@ -30,14 +34,19 @@ def __init__(self, api_key, app_key):
             setattr(self, method, getattr(statsd, method))
 
 
+class DataDogClientResource(ConfigurableResourceFactory[DataDogClient]):
+    api_key: str = Field(description="Datadog API key")
```

Comment:
nit: add description as docstring (could just use the description from `@resource` below)

could also consider linking to docs about API/app key, we do that inconsistently among other resources (https://docs.datadoghq.com/account_management/api-app-keys/)